scripts/globals/mobskills/diffusion_ray.lua

```diff
@@ -0,0 +1,21 @@
+---------------------------------------------
+-- Difusion Ray
+-- Description: Deals damage to enemies within a fan-shaped area originating from the caster.
+-- Type: Magical Light (Element)
+---------------------------------------------
+require("scripts/globals/monstertpmoves")
+require("scripts/globals/status")
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    return 0
+end
+
+function onMobWeaponSkill(target, mob, skill)
+
```

Comment:
My OCD humbly requests:
```'lua
function onMobWeaponSkill(target, mob, skill)
    local dmgmod = MobBreathMove(mob, target, 0.2, 0.65, dsp.magic.ele.LIGHT, 500)
    local dmg = MobFinalAdjustments(dmgmod,mob,skill,target,dsp.damageType.BREATH,dsp.attackType.LIGHT,MOBPARAM_IGNORE_SHADOWS)
    target:takeDamage(dmg, mob, dsp.attackType.BREATH, dsp.damageType.LIGHT)

    return dmg
end
```
So that it,s line breaks are consistent with other files. Doesn't effect its mergability or its usability, just makes my OCD keep noticing it.

Reply:
My own OCD gets me to format it this way. I can change it up though